geomstats/geometry/complex_poincare_disk.py

```diff
@@ -0,0 +1,360 @@
+"""The complex Poincaré disk manifold.
+
+The Poincaré complex disk is a representation of the Hyperbolic space of dimension 2.
+
+Lead author: Yann Cabanes.
+
+References
+----------
+.. [Cabanes2022] Yann Cabanes. Multidimensional complex stationary
+    centered Gaussian autoregressive time series machine learning
+    in Poincaré and Siegel disks: application for audio and radar
+    clutter classification, PhD thesis, 2022
+.. [JV2016] B. Jeuris and R. Vandebril. The Kahler mean of Block-Toeplitz
+    matrices with Toeplitz structured blocks, 2016.
+    https://epubs.siam.org/doi/pdf/10.1137/15M102112X
+.. [Yang2013] Marc Arnaudon, Frédéric Barbaresco and Le Yang.
+    Riemannian medians and means with applications to radar signal processing,
+    IEEE Journal of Selected Topics in Signal Processing, vol. 7, no. 4, pp. 595-604,
+    Aug. 2013, doi: 10.1109/JSTSP.2013.2261798.
+    https://ieeexplore.ieee.org/document/6514112
+"""
+
+import geomstats.backend as gs
+from geomstats.geometry.base import ComplexOpenSet
+from geomstats.geometry.complex_riemannian_metric import ComplexRiemannianMetric
+from geomstats.geometry.hermitian import Hermitian
+
+
+class ComplexPoincareDisk(ComplexOpenSet):
+    """Class for the complex Poincaré disk.
+
+    The Poincaré disk is a representation of the Hyperbolic
+    space of dimension 2. Its complex dimension is 1.
+
+    Parameters
+    ----------
+    scale : float
+        Scale of the complex Poincaré metric.
+        Optional, default: 1.
+    """
+
+    def __init__(self, scale=1.0, **kwargs):
```

Comment:
shouldn't `scale` should be only a metric argument?

I think this applies to other spaces, but I only realized it now.

I'll also soon propose to remove it completely from the manifold and find a different way to implement scale.

Reply:
Hello @lpereira95, 
Yes, `scale` is only a metric argument.
I think that it is nice to have the scale parameter defined when we instanciate `Manifold.`
What would you suggest?
This will be the subject of another PR, right?